routes/api/v1/cgTranscripts.js

```diff
@@ -0,0 +1,110 @@
+var express = require("express");
+var router = express.Router();
+var mongoose = require("mongoose");
+var fq = require("fuzzquire");
+var users = fq("users");
+var config = fq("config");
+var path = require("path");
+var fs = require("fs");
+let appRoot = require("app-root-path");
+var errorFields = [];
+var cgTranscriptsModel = require("../../../schemas/cgTranscripts");
+
+let authenticate = function(req, res, next) {
+  if (true) {
+    return next();
+  } else {
+    res.status(401).json({
+      error: true,
+      message: "Uable to authenticate"
+    });
+    return;
+  }
+};
+
+let checkErrors = function(req, res, next) {
+  if (errorFields.length > 0) {
+    res.status(400).json({
+      error: true,
+      message: `Invalid fields ${errorFields}`
+    });
+    return true;
+  }
+  return false;
+};
+
+router.all("/", function(req, res, next) {
+  errorFields = []; // The error fields must be reset for every query
+  next();
+});
+
+router.get("/test", authenticate, function(req, res) {
+  res.json({ message: "API is online." });
```

Comment:
Shouldn't a `status` be also sent(say `200`)

Reply:
200 is always sent unless there is an error.